app/models/middleware_server.rb

```diff
@@ -33,6 +37,21 @@ def evaluate_alert(alert_id, event)
   end
 
   def in_domain?
-    !middleware_server_group.nil?
+    middleware_server_group
```

Comment:
`nil == false`, and everything else besides `false` is truthy. Predicate methods do not need to return true/false explicitly, just a truthy and a falsey value. This is common pratice in ruby.